docs/core/tools/dotnet-watch.md

```diff
@@ -208,6 +208,30 @@ More files can be watched by adding items to the `Watch` group. For example, the
 </ItemGroup>
 ```
 
+## Exclude files and folders using DefaultItemExcludes
+
+Starting in .NET 10, use the `DefaultItemExcludes` property to exclude entire folders or file patterns from being watched by `dotnet watch`. This approach is useful when you want to exclude files that aren't relevant to compilation or files that trigger unwanted restarts or reloads.
+
+For example, files in the `App_Data` folder of ASP.NET Core applications might change while the app runs, causing unnecessary page reloads. Exclude this folder from being watched:
+
+```xml
+<PropertyGroup>
+  <DefaultItemExcludes>$(DefaultItemExcludes);**/App_Data/**</DefaultItemExcludes>
+</PropertyGroup>
+```
+
+Exclude multiple patterns by separating them with semicolons:
+
+```xml
+<PropertyGroup>
+  <DefaultItemExcludes>$(DefaultItemExcludes);**/App_Data/**;**/temp/**;**/*.log</DefaultItemExcludes>
+</PropertyGroup>
+```
+
+The `DefaultItemExcludes` property affects all default item types (like `Compile` and `EmbeddedResource`), while the `Watch="false"` attribute provides finer control over specific files or project references.
+
+For more information, see the [DefaultItemExcludes reference](../project-sdk/msbuild-props.md#defaultitemexcludes).
```

Comment:
We should probably also add a note to https://review.learn.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props?branch=pr-en-us-50550#defaultitemexcludes that it affects `dotnet watch`.